service/service.py

```diff
@@ -306,6 +311,45 @@ def wishlists_reset():
         app.logger.info('Request to remove all wishlists from database')
         return make_response('', status.HTTP_204_NO_CONTENT)
 
+######################################################################
+#  PATH: /wishlists/{id}/items
+######################################################################
+@api.route('/wishlists/<wishlist_id>/items')
+@api.param('wishlist_id', 'The Wishlists unique ID number')
+class ProductCollection(Resource):
+    ######################################################################
+    # QUERY AND LIST WISHLIST ITEM
+    ######################################################################
+    @api.doc('list_wishlist_item')
+    @api.expect(wishlist_item_args, validate=True)
+    @api.response(404, 'No wishlist item found.')
+    @api.marshal_list_with(wishlistProduct_model)
+    def get(self, wishlist_id):
+        """ Query a wishlist items from URL """
+        app.logger.info('Querying Wishlist items')
+        wishlist = Wishlist.find(wishlist_id)
+        if not wishlist:
+            api.abort(404, "Wishlist was not found.")
+        wishlist_id = request.args.get('wishlist_id')
```

Comment:
I have 2 concerns here:
1) I think we have a bug in our code. Can we use `wishlist_id` from the request param? It doesn't make sense to call an api with a particular `wishlist_id` and allow some other `wishlist_id` to be set in the query params. We might need to fix some tests for this as well.
2) Another is that the `wishlist_id` hasn't been mentioned in the model. I think the previous tests were failing due to `required=True`. We could reintroduce that argument in `wishlist_item_args`, not change any tests right now, but change them in a later PR.

Reply:
I've tried to use the `wishlist_id` from the args, but when I execute the docs, it just keeps loading without any errors or logs.

Reply:
Do you mean the args in `wishlist_item_args`? Or the query param args? Or the ones we pass in the api endpoint?

Reply:
In `wishlist_item_args`, when I defined it in the model, and use `wishlist_id = request.args.get('wishlist_id')` for all `wishlist_id`.

Reply:
even with `required=False`?

Reply:
Yes

Reply:
okay. That's interesting. So do you want to just go ahead and merge this and we do the fixing of the tests as a cleanup task?

Reply:
How do you want to change the tests? Like checking if the same wishlist_id is being used?

Reply:
I was thinking not considering wishlist id from the query params (`request.args.get`). Just the url param should suffice.

Reply:
Ok I just changed the `wishlist_id` in service.py. I'm not sure if there are tests to be changed, but I will take a look tomorrow.

Reply:
Thanks. All Tests pass. There shouldn't be anything necessary to change then